l10n_us_gaap/tests/test_base_us.py

```diff
@@ -0,0 +1,16 @@
+from odoo.tests.common import TransactionCase
+from odoo.tests import tagged
+
+
+@tagged('post_install', '-at_install')
+class TestChartAcount(TransactionCase):
+
+    def test_basic(self):
+        chart_us = self.env.ref('l10n_us_gaap.l10n_us_gaap_chart_template')
+        chart_us.try_loading_for_current_company()
+        bank_account = self.env['account.account'].search([('code', '=', '111101')])
+        liquidity_transfer = self.env['account.account'].search([('code', '=', '111901')])
+        finished_goods = self.env['account.account'].search([('code', '=', '133000')])
+        self.assertEqual(bank_account.name, 'Bank')
```

Comment:
These line is failed. Can you check these.

![Screenshot_2019-07-15 Job #219 3 - OCA l10n-usa - Travis CI](https://user-images.githubusercontent.com/14229503/61225531-640e9900-a73e-11e9-9e3a-200d2dd92d0e.png)